azkaban-common/src/main/java/azkaban/executor/ExecutorHealthChecker.java

```diff
@@ -136,14 +136,17 @@ void checkExecutorHealth() {
       Optional<ExecutorManagerException> healthcheckException = Optional.empty();
       Map<String, Object> results = null;
       try {
-        // Todo jamiesjc: add metrics to monitor the http call return time
+        long pingTime = System.currentTimeMillis();
         results = this.apiGateway
             .callWithExecutionId(executor.getHost(), executor.getPort(),
                 ConnectorParams.PING_ACTION, null, null, null);
+        pingTime = System.currentTimeMillis() - pingTime;
+        logger.info("Got ping response from " + executorDetailString(executor)
```

Comment:
Any thoughts around how frequently these log messages will be printed? 
(Just to make sure the logs are not excessively filled up with these,  AFAIR the healtcheck was not happen too frequently for this to be a concern though)

Reply:
Its every 5 mins from what I see. Hope thats okay.